config/routes.rb

```diff
@@ -1,10 +1,37 @@
 Rails.application.routes.draw do
+  get 'general_shopping_lists/index'
+  get 'recipe_foods/index'
+  get 'recipe_foods/show'
+  get 'recipe_foods/new'
+  get 'recipe_foods/create'
+  get 'recipe_foods/edit'
+  get 'recipe_foods/update'
+  get 'recipe_foods/destroy'
+  get 'public_recipes/index'
+  devise_for :users
```

Comment:
- [ ] An error is thrown in an attempt to logout. Let's ensure to fix it:

![image](https://github.com/geekelo/domins-recipe_app/assets/12745474/e0fb18ca-4889-4ae7-b376-feaedeea689f)